scripts/package/fpsci_packager.py

```diff
@@ -0,0 +1,68 @@
+# This script generates a packaging script based on the files used by the last successful run of the program.
+# You need to run the generated script to create the distribution directory, then zip that up to send to people.
+
+# Name of the packaging script to generate
+outputScript = 'package.sh'
+
+# Path of the log.txt file to use
+inputLog = 'data-files/log.txt'
+
+# The list of files that should always be excluded
+excludeList = ['log.txt', 'keymap.Any', 'systemconfig.Any']
+# The list of files that should be excluded for release builds, but may want to be included for specific experiment distributions
+genericExcludeList = ['experimentconfig.Any', 'startupconfig.Any', 'userconfig.Any', 'userstatus.Any']
+
+# List of paths to search for to merge into the same distribution directory
+basePath = ['data-files', 'game', 'common']
+
+# Distribution path - where the output script will generate the distribution
+distPath = 'dist/'
+
+
+from datetime import datetime
+
+if __name__ == '__main__':
+    # The log file which lists the files used
+    log = open(inputLog)
+
+    # The package script to emit
+    packageScript = open(outputScript, 'w')
+    packageScript.writelines(['#!/bin/bash\n', '# Autogenerated ' + datetime.now().strftime('%Y %m %d %H:%M:%S\n')])
+    packageScript.write('mkdir -p ' + distPath + '\n')
+
+    # Set up copy of .exe
+    packageScript.write('cp Build/FirstPersonScience-x64-Release/FirstPersonScience.exe ' + distPath + '\n')
+
+    # main loop, need to ignore lines before the file list begins
+    beforeFiles = True
+    for line in log.readlines():
+        # In the file list, add this file to what we want to copy
+        if not beforeFiles and line != '\n':
+            filename = line.strip()
+            basename = filename.split('/')[-1]
+            # Exclude the lists of files we don't want to package
+            if basename in excludeList or basename in genericExcludeList:
+                continue
+            
+            # Absolute path given
+            if filename.startswith('C:/') or filename.startswith('c:/'):
```

Comment:
This may create issues for folks packaging from a build in a drive with a different name. Can go as-is and be updated later. Could use that the 2nd-3rd character is `:/` to detect absolute paths.

Reply:
Good suggestion, let's do that.

I don't like being so windows-specific, but that's just where we are right now.